nativescript-angular/router/ns-location-strategy.ts

```diff
@@ -302,23 +303,21 @@ export class NSLocationStrategy extends LocationStrategy {
     }
 
     public _finishBackPageNavigation() {
-        routerLog("NSLocationStrategy.finishBackPageNavigation()");
         if (!this._isPageNavigationBack) {
-            throw new Error("Calling endGoBack while not going back.");
+            routerError("Attempted to call endGoBack while not going back.");
+            return;
         }
+        log("NSLocationStrategy.finishBackPageNavigation()");
         this._isPageNavigationBack = false;
     }
 
     public _isPageNavigatingBack() {
         return this._isPageNavigationBack;
     }
 
-    public _beginModalNavigation(frame: Frame): void {
-        routerLog("NSLocationStrategy._beginModalNavigation()");
-
-        let { cachedFrameRootOutlet } = this.frameService.findFrame(frame);
-
-        const lastState = this.peekState(cachedFrameRootOutlet || this.currentOutlet);
+    public _beginModalNavigation(): void {
```

Comment:
Shouldn't remove the `frame: Frame` param. Also, some logic from the method is removed. I guess something gone went while updating the branch?